.babelrc.js

```diff
@@ -43,7 +44,7 @@ const config = {
 			{ async: isCalypsoClient && codeSplit },
 		],
 		'@babel/plugin-proposal-export-default-from',
-		'@babel/transform-runtime',
+		[ '@babel/transform-runtime', { regenerator: false } ],
```

Comment:
I'm pretty sure we can drop this completely if we're using the polyfill

Reply:
Seems happy without, so pushed the change here.